exchanges/bithumb/bithumb.go

```diff
@@ -173,32 +182,41 @@ func (b *Bithumb) GetAllTickers() (map[string]Ticker, error) {
 		return nil, err
 	}
 
-	if response.Status != noError {
+	if response.Status != "0000" {
 		return nil, errors.New(response.Message)
 	}
 
 	result := make(map[string]Ticker)
 	for k, v := range response.Data {
-		if k == "date" {
+		data, ok := v.(map[string]interface{})
```

Comment:
This can be rewritten much more simply:
```
type Response struct {
		ActionStatus
		Data map[string]json.RawMessage
	}

	response := Response{}
	path := fmt.Sprintf("%s%s%s", b.APIUrl, publicTicker, "all")

	err := b.SendHTTPRequest(path, &response)
	if err != nil {
		return nil, err
	}

	if response.Status != "0000" {
		return nil, errors.New(response.Message)
	}

	result := make(map[string]Ticker)
	for k, v := range response.Data {
		var helloMoto Ticker
		err := common.JSONDecode(v, &helloMoto)
		if err != nil {
			log.Error(err)
		}
		result[k] = helloMoto
	}
	return result, nil
```

Reply:
Perhaps having an if for the date parsing, this way still reduces the complexity and doesn't ignore error checking
```if k == "date" {
// continue because its not used
} else {
 // ticker parse
}
```

Reply:
That's already handled in this because v.(type) == string
```go
data, ok := v.(map[string]interface{})
		if !ok {
			continue
		}
```

Reply:
Okay I updated to what you suggested